core/src/test/java/org/mobilitydata/gtfsvalidator/notice/NoticeContainerTest.java

```diff
@@ -165,4 +168,87 @@ public void addAll() {
     assertThat(c1.getValidationNotices()).containsExactly(n1, n2);
     assertThat(c1.getSystemErrors()).containsExactly(e1, e2);
   }
+
+  @Test
+  public void exportNoticesSchema() throws IOException {
```

Comment:
This test is unstable, it fails on my machine. Did you consider to run it several times?


```
diff:
    @@ -1,5 +1,5 @@
     {
    -  "AnotherTestValidationNotice": [
    +  "SomeTestValidationNotice": [
         {
           "name": "filename",
           "type": "VARCHAR"
    @@ -17,27 +17,11 @@
           "type": "BLOB"
         },
         {
    -      "name": "otherFieldValue",
    -      "type": "DOUBLE"
    -    },
    -    {
    -      "name": "sampleDate",
    +      "name": "severityLevel",
           "type": "BLOB"
    -    },
    -    {
    -      "name": "sampleTime",
    -      "type": "BLOB"
    -    },
    -    {
    -      "name": "sampleColor",
    -      "type": "VARCHAR"
    -    },
    -    {
    -      "name": "SeverityLevel",
    -      "type": "BLOB"
         }
       ],
    -  "SomeTestValidationNotice": [
    +  "AnotherTestValidationNotice": [
         {
           "name": "filename",
           "type": "VARCHAR"
    @@ -55,8 +39,24 @@
           "type": "BLOB"
         },
         {
    -      "name": "severityLevel",
    +      "name": "otherFieldValue",
    +      "type": "DOUBLE"
    +    },
    +    {
    +      "name": "sampleDate",
           "type": "BLOB"
    +    },
    +    {
    +      "name": "sampleTime",
    +      "type": "BLOB"
    +    },
    +    {
    +      "name": "sampleColor",
    +      "type": "VARCHAR"
    +    },
    +    {
    +      "name": "SeverityLevel",
    +      "type": "BLOB"
         }
       ],
       "NestedTestValidatorNotice": [
	at org.mobilitydata.gtfsvalidator.notice.NoticeContainerTest.exportNoticesSchema(NoticeContainerTest.java:177)
```

Reply:
I modified the code so that the keys of the json file are sorted alphabetically. I ran it several times on my machine and it seems to work. Could you confirm the new behavior on your side please @aababilov?

Reply:
Thanks for fixing!